README.md

```diff
@@ -88,34 +108,52 @@ yum install -y epel-release
 yum install -y readline-devel leveldb-devel libffi-devel gcc-c++ redhat-rpm-config gcc python-devel openssl-devel
 ```
 
-### For all of these, make sure that the `Chains` directory in your project has the proper write permissions
+#### Windows
 
-##### Windows
+Help needed. Installing the Python package plyvel seems to require C++ compiler
+support tied to Visual Studio and libraries.
 
-Not sure. Installing the Python package plyvel seems to require C++ compiler support tied to Visual Studio and libraries.
+### Python 3.5
 
--------------------
+neo-python is currently only compatible with **Python 3.5** (due to its `byteplay3` dependency).
 
-### Virtual Environment
+On *nix systems, install Python 3.5 via some package manager, or download an installation package
+from the [official homepage](https://www.python.org/downloads/release/python-354).
 
-Now navigate into the project, make a Python 3 virtual environment and activate
-it via
+#### Virtual Environment
+
+It is recommended to put all project dependencies into its own virtual environment,
+this way we don't pollute the global installation which could lead to version conflicts.
 
 ```
 python3.5 -m venv venv
 source venv/bin/activate
 ```
 
-Then install the requirements:
+Now let's install neo-python's dependencies:
 
 ```
 pip install -U setuptools pip wheel
 pip install -e .
 ```
 
+Tip: using [autoenv](https://github.com/kennethreitz/autoenv) to load the configuration
```

Comment:
autoenv is inactive and he says on the github page to try another project. i would not mention it here. people working with this should be able to run `python3.5 -m venv venv`